components/blocks/autofunction.js

```diff
@@ -214,6 +231,21 @@ const Autofunction = ({
       }
       rows={args.length ? args : null}
       addtionalClass="full-width"
+    />
+  );
+
+  returnBody = (
```

Comment:
@snehankekre and @tvst: Do you think it makes sense to have the returns in a separate table? Or should it be part of the above one?

The reason I'm asking this is the fact it creates a [visual separation between the table contents](https://share.getcloudapp.com/qGurPjnz) and I'm not sure that's what we want

Reply:
> Do you think it makes sense to have the returns in a separate table? Or should it be part of the above one?

@imjuangarcia No, it shouldn't be a separate table and should be a part of the above one. I should have mentioned above that I created a new table for the returns because I didn't know the JS needed to add it to the above table. :sweat_smile: 

My janky implementation should not be merged IMO. I'd love your and/or Thiago's help with adding the returns info to the original table (perhaps as an additional `<tbody></tbody>`). :point_left: This is what I didn't know how to do in JS.

Reply:
@snehankekre Got it, thanks for letting me know! Just fixed this issue on a7a1616, using a `<tfoot>` element. Let me know what you think!

Reply:
@imjuangarcia Looks splendid! :tada:  After adding the missing footers on 3b4fbb2, I think we're good to merge, unless @tvst or @randyzwitch have objections.